src/components/Results/ArticleSummary.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Grid, Typography, makeStyles } from "@material-ui/core/";
+import "../../App.css";
+import TruncateMarkup from "react-truncate-markup";
+//Truncate  & Textfit need to be updated, throwing unsafe errors
+
+const useStyles = makeStyles({
+  ArticleSummaryText: {
+    fontFamily: "Roboto, sans-serif",
+    padding: "10px",
+    textAlign: "left",
+  },
+  ArticleTitle: {
+    fontSize: "20px",
+    fontWeight: "700",
+    marginBottom: "18px",
+  },
+  ArticleURL: {
+    fontSize: "12px",
+    marginBottom: "10px",
+  },
+  SummaryText: {
+    marginBottom: "12px",
+    fontSize: "16px",
+  },
+});
+
+function ArticleSummary() {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Grid
+        container
+        justify="flex-start"
+        direction="column"
+        className={classes.ArticleSummaryText}
```

Comment:
ArticleText in place of summary as the analysis is on article

Reply:
This is the className for the container box of the all  the text. Is that what you mean?